README.md

```diff
@@ -39,6 +42,8 @@ Imagewoof is a subset of 10 classes from Imagenet that aren't so easy to classif
 - [320 px download](https://s3.amazonaws.com/fast-ai-imageclas/imagewoof2-320.tgz); 
 - [160 px download](https://s3.amazonaws.com/fast-ai-imageclas/imagewoof2-160.tgz). 
 
+The dataset also comes with a CSV file with 1%, 5%, 25%, and 50% of the labels randomly changed to an incorrect label. More information about the noisy labels are provided in the "noisy_imagenette" folder.
```

Comment:
consider renaming the folder to `noisy_labels` since your thing applies to both imagenette and imagwoof

Reply:
@hamelsmu Good point, thanks for the catch! I have fixed it now.